src/main/java/endergeticexpansion/core/events/SRCompatEvents.java

```diff
@@ -0,0 +1,37 @@
+package endergeticexpansion.core.events;
+
+import endergeticexpansion.common.entities.booflo.EntityBooflo;
+import endergeticexpansion.common.entities.booflo.EntityBoofloAdolescent;
+import endergeticexpansion.common.entities.booflo.EntityBoofloBaby;
+import endergeticexpansion.core.EndergeticExpansion;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.util.ResourceLocation;
+import net.minecraftforge.event.entity.living.PotionEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.registries.ForgeRegistries;
+
+/**
+ * @author Voliant
+ * Events for compatibility with the Savage & Ravage mod.
+ */
+@Mod.EventBusSubscriber(modid = EndergeticExpansion.MOD_ID)
+public class SRCompatEvents {
+    @SubscribeEvent
+    public static void onPotionExpire(PotionEvent.PotionExpiryEvent event) {
+        LivingEntity affected = event.getEntityLiving();
+        boolean isBabyEffect = event.getPotionEffect().getPotion() == ForgeRegistries.POTIONS.getValue(new ResourceLocation("savageandravage:shrinking"));
+        if(isBabyEffect || event.getPotionEffect().getPotion() == ForgeRegistries.POTIONS.getValue(new ResourceLocation("savageandravage:growth"))) {
+            if(!isBabyEffect && affected instanceof EntityBoofloBaby) ((EntityBoofloBaby)affected).growUp();
+            if(affected instanceof EntityBoofloAdolescent) {
+                if (isBabyEffect) {
```

Comment:
This line here is the last nitpick 😛

Reply:
Are you suggesting to add spacing between the if statements? Sorry if I’m missing something, I’m in bed on mobile

Reply:
Ok i just went on the pc and i actually don't know what you're referring to

Reply:
Most of your if statements have no spacing between them, so all of them should do that.

Reply:
I see. Changing now.

Reply:
Not what I was referring to lol...

Reply:
Well, what were you referring to? There's spaces around the && and ||, and that's the only other thing I can think of. Can you please just write out what changes you want me to make?